src/test/regress/sql/prepare.sql

```diff
@@ -65,6 +65,11 @@ SELECT * FROM q5_prep_results;
 CREATE TEMPORARY TABLE q5_prep_nodata AS EXECUTE q5(200, 'DTAAAA')
     WITH NO DATA;
 SELECT * FROM q5_prep_nodata;
+PREPARE p AS SELECT 1 i;
```

Comment:
Add empty string and short comment about test.

Reply:
added